src/NzbDrone.Core/ImportLists/Trakt/User/TraktUserParser.cs

```diff
@@ -0,0 +1,71 @@
+using System.Collections.Generic;
+using System.Linq;
+using NzbDrone.Common.Extensions;
+using NzbDrone.Common.Serializer;
+using NzbDrone.Core.Parser.Model;
+
+namespace NzbDrone.Core.ImportLists.Trakt.User
+{
+    public class TraktUserParser : TraktParser
+    {
+        private readonly TraktUserSettings _settings;
+        private ImportListResponse _importResponse;
+
+        public TraktUserParser(TraktUserSettings settings)
+        {
+            _settings = settings;
+        }
+
+        public override IList<ImportListItemInfo> ParseResponse(ImportListResponse importResponse)
+        {
+            _importResponse = importResponse;
+
+            var listItems = new List<ImportListItemInfo>();
+
+            if (!PreProcess(_importResponse))
+            {
+                return listItems;
+            }
+
+            var jsonResponse = new List<TraktSeriesResource>();
```

Comment:
I think `traktSeries` makes more sense (since it's a list of series) than `jsonResponse`